src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,111 @@
+@use '../utils/variables' as *;
+@use '../utils/mixins' as *;
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding: 16px;
+
+  font-family: Roboto, sans-serif;
+
+  @include transition;
```

Comment:
Ensure that the `transition` mixin is defined in the `mixins` file. If it's not defined, this could lead to a compilation error.